internal/app/recordtester/recordtester_app.go

```diff
@@ -363,10 +363,13 @@ func (rt *recordTester) Start(fileName string, testDuration, pauseDuration time.
 		if code, err := checkPlaybackInfo(assets[0].PlaybackID, rt.lapi, lapiNoAPIKey); err != nil {
 			return code, err
 		}
+
+		rt.lapi.DeleteAsset(assets[0].ID)
```

Comment:
shouldn't this be a `defer` call earlier, before one of the above `return` statements?